### PR TITLE
Update modules

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

Update modules to their latest version and associated changes in the subworkflows. 
One significant change is that there is no need anymore to supply vcfanno_resources as a directory. Instead, the absolute path of files and indices will have to be provided as a text file which will then be parsed in the pipeline. This is to accommodate the latest changes to the structure of vcfanno input channels. 

*This should also fix the linting errors we are seeing over at https://github.com/nf-core/raredisease/pull/258

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by Lucpen — APPROVED** [review]
LGTM!

File: subworkflows/local/qc_bam.nf [hunk 1]
```diff
@@ -14,6 +14,7 @@ workflow QC_BAM {
     take:
         bam              // channel: [ val(meta), path(bam) ]
         bai              // channel: [ val(meta), path(bai) ]
+        bam_bai
```
Comment (jemten) [inline subworkflows/local/qc_bam.nf:17]: We could update the mosdepth part to take the mixed bam bai channel as well

**Review by jemten — COMMENTED** [review]
Looks good. Had one comment about mosdepth and potentially using the bam_bai channel

File: subworkflows/local/qc_bam.nf (hunk 1 above)
Comment (ramprasadn) [reply subworkflows/local/qc_bam.nf:17]: True I will create a note and fix it in another PR

**Pull request merged**